components/homme/src/share/prim_driver_base.F90

```diff
@@ -1106,10 +1116,12 @@ subroutine prim_step(elem, hybrid,nets,nete, dt, tl, hvcoord, compute_diagnostic
     ! ===============
     call t_startf("prim_step_dyn")
     call prim_advance_exp(elem, deriv1, hvcoord,   &
-         hybrid, dt, tl, nets, nete, compute_diagnostics)
+         hybrid, dt, tl, nets, nete, compute_diagnostics, &
+	 single_column)
```

Comment:
See comment for `components/homme/src/pese/prim_advance_mod.F90`, I think maybe you meant to have consistency across all versions of HOMME for the `prim_advance_exp` subroutine.  Does this mean that single_column isn't yet supported for the `pese` version?